packages/hint-meta-charset-utf-8/src/hint.ts

```diff
@@ -98,15 +124,32 @@ export default class MetaCharsetUTF8Hint implements IHint {
             const isMetaElementFirstHeadContent = (/^<head[^>]*>\s*<meta/).test(headElementContent);
 
             if (!isCharsetMetaFirstHeadElement || !isMetaElementFirstHeadContent) {
-                context.report(resource, getMessage('metaElementFirstThing', context.language), { element: charsetMetaElement });
+
+                const severity = firstHeadElement.getLocation().endOffset || 0 <= 1024 ?
```

Comment:
Guess it's about type to upgrade to TypeScript 3.7. Ready for that nullish coalescing! 😉

Reply:
Yes please 😉

https://twitter.com/molant/status/1193293388822401025?s=20